mantidproject/mantid
# Add error cols for some logs in muon GUI results table

**Description of work.**
The results table tab in the muon GUI would not include any errors for the log values. This PR fixes that. 




**To test:**


Turn on the model analysis tab and plot 
Open Mantid
Open muon analysis
Load some data e.g. HIFI 84447-8
Go to the fitting tab
Click simultaneous and add a fit function (doesnt matter which)
Go to sequential tab and click "fit all"
Go to the results tab
In the top log table select

- run number (will have an error of 0)
- run start (wont have an error as its a string)
- beamlog count rate (will have a numerical error)
- detector temp 41 (will also have a numerical error)

Click the "results table" button at the bottom of the tab
Check the results table

On the model analysis tab, if you set the y axis to a log with errors they will be shown on the plot. The x errors are not given at present, see https://github.com/mantidproject/mantid/issues/32914

Fixes #32820. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
